h2o-core/src/main/java/water/persist/PersistH2O.java

```diff
@@ -0,0 +1,90 @@
+package water.persist;
+
+import org.apache.commons.io.IOUtils;
+import water.Key;
+import water.util.FrameUtils;
+import water.util.Log;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class PersistH2O extends EagerPersistBase {
+
+    public static final String SCHEMA = "h2o";
+    public static final String PREFIX = SCHEMA + "://";
+    private static final String H2O_RESOURCE_PATH = "/extdata/";
+
+    private static final List<String> CONTENTS; 
+
+    static {
+        List<String> contents = Collections.emptyList();
+        try {
+            contents = readExtDataContents();
+        } catch (IOException e) {
+            Log.trace(e);
+        }
+        CONTENTS = Collections.unmodifiableList(contents);
+    }
+   
+    @Override
+    public void importFiles(String path, String pattern,
+            /*OUT*/ ArrayList<String> files, ArrayList<String> keys, ArrayList<String> fails, ArrayList<String> dels) {
+        try {
+            URL resourceURL = pathToURL(path);
+            if (resourceURL == null) {
+                fails.add(path);
+                Log.err("Resource '" + path + "' is not available in H2O.");
+                return;
+            }
+            Key<?> destination_key = FrameUtils.eagerLoadFromURL(path, resourceURL);
+            files.add(path);
+            keys.add(destination_key.toString());
+        } catch (Exception e) {
+            Log.err("Loading from `" + path + "` failed.", e);
+            fails.add(path);
+        }
+    }
+
+    static URL pathToURL(String path) {
+        String[] pathItems = path.split(":", 2);
```

Comment:
```suggestion
        String[] pathItems = path.split("://", 2);
```
it would make the actual paths below a bit less "lucky":
```java
URL resource = PersistH2O.class.getResource(H2O_RESOURCE_PATH + pathItems[1]+".csv");
# e.g. getResource("/exdata/iris.csv")
# instead of getResource("/exdata///iris.csv")
```
I know that both work but normalized paths look nicer.

Reply:
ok, implemented